docs/user_guide/3_installing_the_software.rst

```diff
@@ -0,0 +1,83 @@
++---------------+------------------------------------------------------------------------------------------------------------+
+| Hard Drive    | Fast HDD or SSD as laptop HDD are very slow (500 GB SSD recommended)                                       |
++---------------+------------------------------------------------------------------------------------------------------------+
+| Graphics Card | Nvidia GPU (optional)                                                                                      |
++---------------+------------------------------------------------------------------------------------------------------------+
+| LAN           | A spare LAN port to connect the Hand (even with a USB to LAN adaptor)                                      |
++---------------+------------------------------------------------------------------------------------------------------------+
+| OS            | Ubuntu 20.04 Noetic (Active development), 18.04 Melodic, 16.04 Kinetic or 14.04 Indigo for older releases. |
++---------------+------------------------------------------------------------------------------------------------------------+
+
+The most important one is to have a fast HDD or an SSD.
+
+We have created a one-liner that is able to install Docker, download the docker image and create a new container for you.
+It will also create desktop icons, one to start the container, one to launch the hand driver on the control box and one to save the log files locally.
+To use it, you first need to have a PC with Ubuntu installed on it, then follow the next steps.
+
+Get ROS Upload login credentials
+---------------------------------
+
+If you want to upload technical logged data (ROS logs, backtraces, crash dumps etc.) to our server and notify the Shadow's software team to investigate your bug, then you need to enable logs uploading in the one-liner.
+In order to use this option you need to obtain a unique upload key. It can be found in the delivering instructions or by emailing sysadmin@shadowrobot.com. When you receive the key you can use it when running the one-liner installation tool.
+To enable the logs uploading you need to add the command line option ``--read-secure customer_key`` to the one-liner.
+After executing the one-liner, it will prompt you to enter your "Secure data input for customer_key". Please copy and paste here your key.
+
+Run the one-liner
+------------------
+
+The one-liner will install Docker, pull the image from Docker Hub, and create and run a container with the parameters specified. In order to use it, follow these instructions:
+
+  1. Connect the ethernet between the NUC-CONTROL and the new PC using the instructions above
+  2. Power on the new PC
+  3. Connect an ethernet cable providing external internet connection to the back of the new PC
+  4. Power on the NUC-CONTROL
+  5. Install the hand software on the new PC by running the following on a terminal (Ctrl+Alt+T):
+
+ROS Melodic (Recommended) for a Right Hand:
```

Comment:
Shouldn't `Noetic` be recommended now?

Reply:
Added noetic now and made it the recommended option.